test/image/mocks/axes_range_manual.json

```diff
@@ -42,13 +46,18 @@
         1,
         2,
         3,
+        -3000000,
+        3e100,
+        1e12,
+        -1e11,
         4,
         5,
         6,
         7,
         8
       ],
-      "type": "scatter"
+      "type": "scatter",
+      "mode": "lines"
```

Comment:
nw has problems with displaying the markers for this trace if you include them: you get the axes and gridlines but no traces drawn at all. If you drop the exponent 100 to 12 or less, it works again. Clipping markers should be easier than lines, but I opted to leave that for another time. If it's only nw that cares, perhaps image-exporter will moot the issue - but it bears a little more investigation -> #2061

Reply:
BTW I modified this mock because, while `ultra_zoom` does test the functionality, I wanted to cover a few more cases with the image tests. But [scatter_test](https://github.com/plotly/plotly.js/pull/2060/files#diff-88806d1982a87e90bee0165cdabd5b9a) covers many many more cases.

Reply:
using the new image exporter and :hocho: that `"mode": "lines",` line, we get:

![image](https://user-images.githubusercontent.com/6675409/31199804-8d33586a-a926-11e7-81b1-0b6eb3ec1eaf.png)